pkg/azuredisk/azuredisk_v2.go

```diff
@@ -45,6 +45,7 @@ import (
 	"k8s.io/klog/v2"
 	"k8s.io/klog/v2/klogr"
 
+	crd "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
np: Consider `apiextensionsv1` instead of `crd` for consistency in package aliasing.

Reply:
I replace `crd` with `apiext` to be consistent with other packages in `sigs.k8s.io`.